pkg/agent/agent_linux.go

```diff
@@ -70,22 +70,25 @@ func (i *Initializer) prepareOVSBridgeForK8sNode() error {
 	uplinkNetConfig := i.nodeConfig.UplinkNetConfig
 	uplinkNetConfig.Name = adapter.Name
 	uplinkNetConfig.MAC = adapter.HardwareAddr
-	uplinkIPs, err := getAllIPNetsByName(adapter.Name)
+	_, uplinkIPs, uplinkRoutes, err := getInterfaceConfig(adapter.Name)
```

Comment:
I feel we need not to get uplink IPs and Routes here, but can just get them in `PrepareHostInterfaceConnection`.